Sources/Datadog/RUM/Scrubbing/RUMEventsMapper.swift

```diff
@@ -12,14 +12,16 @@ internal typealias RUMResourceEventMapper = (RUMResourceEvent) -> RUMResourceEve
 internal typealias RUMActionEventMapper = (RUMActionEvent) -> RUMActionEvent?
 
 /// The `EventMapper` for RUM events.
-internal struct RUMEventsMapper: EventMapper {
+internal struct RUMEventsMapper {
```

Comment:
👍  This actually makes a lot of sense when I think of upcoming data scrubbing for other features (logging and tracing). Constraining all features to `f(T) -> T?` would not work well, as in APM scrubbing we may want to give users a `Span` along with the `URLRequest` / `HTTPURLResponse` for additional context decoration (e.g. logging the response body in case of error).